presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -184,6 +184,17 @@ disabled if ``connector.num-io-threads-hw-multiplier`` is set to zero.
 
 Whether async data cache is enabled.
 
+``query-data-cache-enabled-default``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type** ``bool``
+* **Default value:** ``true``
+
+If ``true``, SSD cache is enabled by default and is disabled only if
```

Comment:
Is this a config for making cache behavior compatible with java config ? I'm a bit confused about how this should be set along with other options. Please can you elaborate.

Reply:
Yes. In Presto Java, cache is disabled by default and only enabled when the config node_preference is set to SOFT_AFFINITY.  If we set this config to `false`, cpp will be able to behave the same.